.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,102 @@
+name: Bug Report
+description: File a bug report
+labels: ["bug", "triage"]
+assignees:
```

Comment:
I'd find assigning every issue can create loud vociferousness for maintainers. Also, people may find that every bug is assigned and are hesitant to work on the issue. Would it be better to create an automated GitHub project to track unresolved regularly?

Reply:
I was not thinking maintainers would remain assigned to all bugs, but that the maintainers are responsible for triaging bugs and would assign the correct person at that time. I'm also writing a companion document for how to handle bugs.

Reply:
Yeah, I'd agree bugs need to be triaged in a timely manner. I sometimes scan the issue list to see if there is any that is no one looking at. However, triaging issues that people are not familiar with is a massive time consumer. I'd be interested in means to help in the condition.